core/src/test/java/tech/tablesaw/aggregate/AggregateFunctionsTest.java

```diff
@@ -108,6 +110,27 @@ void testGroupBy() {
     assertEquals(13, result.rowCount());
   }
 
+  @Test
+  void testFirstAsString() {
+    StringColumn company = StringColumn.create("Company");
+    StringColumn sector = StringColumn.create("Sector");
+    Table t = Table.create(company, sector);
+    {
```

Comment:
this syntax is confusing to me. I've never seen curly braces used like this before

Reply:
:-) It's just a code block that allows me to reuse `Row r` since they have a different scope. Handy for tests.

Reply:
I'd prefer to avoid since it's a non-standard construct. Probably the cleanest thing to do would make the methods chainable so that we can do:
```
t.appendRow().setString(0, "Google").setString(1, "Technology");
t.appendRow().setString(0, "Facebook").setString(1, "Technology");
```

Reply:
Ha, that is neat!